regexes/bots.yml

```diff
@@ -805,6 +805,38 @@
     name: 'Google Inc.'
     url: 'https://www.google.com/'
 
+- regex: 'Google-Document-Conversion'
+  name: 'Google-Document-Conversion'
+  category: 'Service Agent'
+  url: 'https://support.google.com/drive/answer/176692?hl=en'
+  producer:
+    name: 'Google Inc.'
+    url: 'https://www.google.com/'
+
+- regex: 'GoogleDocs; apps-spreadsheets'
+  name: 'Google Sheets'
+  category: 'Service Agent'
+  url: 'https://workspace.google.com/products/sheets/'
+  producer:
+    name: 'Google Inc.'
+    url: 'https://www.google.com/'
+
+- regex: 'GoogleDocs; apps-presentations'
+  name: 'Google Slides'
+  category: 'Service Agent'
+  url: 'https://workspace.google.com/products/slides/'
+  producer:
+    name: 'Google Inc.'
+    url: 'https://www.google.com/'
+
+- regex: 'GoogleDocs'
```

Comment:
need strict rule 
`GoogleDocs;` or `docs\.google\.com`

it seems to me that if we leave it as it is, in the future we can define applications as bots.